### PR TITLE
Runner for benchmark tests

### PR DESCRIPTION
**Pull request by msenin94**

### CODE REVIEW

File: mk_verificator/benchmarks/engine/cli.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import argparse
```
Comment (dkulykovmirantis) [inline mk_verificator/benchmarks/engine/cli.py]: Where do you use it&

File: mk_verificator/benchmarks/engine/cli.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+import texttable as tt
+
+
+def draw_results_table(shared_memory):
```
Comment (vkhalin) [inline mk_verificator/benchmarks/engine/cli.py]: Why do we need tables here in the framework at all?

File: mk_verificator/benchmarks/engine/runner.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import multiprocessing as mp
```
Comment (vkhalin) [inline mk_verificator/benchmarks/engine/runner.py]: why multiprocessing but not threading?

File: mk_verificator/benchmarks/main.py [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+if __name__ == '__main__':
+    manager = multiprocessing.Manager()
+    shared_memory = manager.dict()
+
+    processes = {}
+
+    scenarios = []
+    if multiprocess:
+        for scenario, scenario_num in enumerate(scenarios):
```
Comment (vkhalin) [inline mk_verificator/benchmarks/main.py]: is there any difference between enumerate and iteritems?

**Comment by vkhalin** [comment]
we need some more explanation about runner and runner in main

File: mk_verificator/benchmarks/engine/runner.py (hunk 3 above)
Comment (msenin94) [inline mk_verificator/benchmarks/engine/runner.py]: let's keep it because of GIL, that's not so critical for us for now and for perspective too

**Pull request merged**